frontend/src/app.css

```diff
@@ -47,4 +47,38 @@
   #year-list .ais-RefinementList-list {
     columns: 5rem auto;
   }
+  .seek-track-slider .track-slider::-webkit-slider-thumb {
+    @apply -mt-1 w-3.5 h-3.5 bg-green-800 rounded-full appearance-none cursor-pointer;
+  }
+  .seek-track-slider .track-slider::-webkit-slider-runnable-track {
+    @apply h-1.5;
+  }
+  .seek-volume-slider .volume-slider::-webkit-slider-thumb {
+    @apply -mt-0.5 w-2.5 h-2.5 bg-green-800 rounded-full appearance-none cursor-pointer;
+  }
+  .seek-volume-slider .volume-slider::-webkit-slider-runnable-track {
+    @apply h-1.5;
+  }
+
+  /* styles for Firefox */
+  .seek-track-slider .track-slider::-moz-range-thumb {
+    @apply -mt-1 w-3.5 h-3.5 bg-green-800 rounded-full appearance-none cursor-pointer;
+  }
+  .seek-track-slider .track-slider::-moz-range-track {
+    @apply h-1.5;
+  }
+  .seek-volume-slider .volume-slider::-moz-range-thumb {
+    @apply -mt-0.5 w-2.5 h-2.5 bg-green-800 rounded-full appearance-none cursor-pointer;
+  }
+  .seek-volume-slider .volume-slider::-moz-range-track {
+    @apply h-1.5;
+  }
```

Comment:
1. Style `input[type="range"]` instead of specific classes. As it is done in https://css-tricks.com/lets-create-a-custom-audio-player/. Let the volume slider be of the same size as the seek slider to avoid specificity.
2. Move this to the `@base` layer, as we are going to have all the `input range` elements look like this.
2. Use CSS nesting. See `algolia.css` for example.

Reply:
1. But we have two different `range/slider`, one for the `Seek` and other one for the `Volume`. That's why we use classes for each `input range`
2. We will use the classes for applying styles for each input. But inside the parent div we can use the `input range`.
3.  Sure i will place the pseudo code nested.

Reply:
1. But they will look identical (except for width), therefore no need adding separate styles for them in the `@base` layer.
2. All the specific styling will be done in HTML using regular Tailwind classes.